hydra-cluster/src/Hydra/Cluster/Faucet.hs

```diff
@@ -106,19 +106,30 @@ returnFundsToFaucet ::
   RunningNode ->
   Actor ->
   IO ()
-returnFundsToFaucet tracer RunningNode{networkId, nodeSocket} sender = do
+returnFundsToFaucet tracer node sender = do
   (faucetVk, _) <- keysFor Faucet
-  let faucetAddress = mkVkAddress networkId faucetVk
+  senderKeys <- keysFor sender
+  returnAmount <- returnFundsToFaucet' tracer node faucetVk senderKeys
+  traceWith tracer $ ReturnedFunds{actor = actorName sender, returnAmount}
 
-  (senderVk, senderSk) <- keysFor sender
+returnFundsToFaucet' ::
+  Tracer IO FaucetLog ->
+  RunningNode ->
+  VerificationKey PaymentKey ->
```

Comment:
This is never different to the `keysFor Faucet`. 

Should: remove this argument